tests/includes/verbose.sh

```diff
@@ -5,13 +5,16 @@ set_verbosity() {
 	case "${VERBOSE}" in
 	1)
 		set -eu
+		set -o pipefail
 		;;
 	2)
 		set -eu
+		set -o pipefail
 		;;
 	11)
```

Comment:
Unrelated: you can't set VERBOSE to 11 with just a flag to main.sh. What is it intended for?

Reply:
I'll remove, it was from a very old method